CHANGELOG.md

```diff
@@ -0,0 +1,109 @@
+# TRILLIAN Changelog
+
+## v1.0.6 - GetLeavesByRange. 403 Permission Errors. Signer Metrics.
+
+Published 2018-02-05 16:00:26 +0000 UTC
+
+A new log server RPC API has been added to get leaves in a range. This is a more natural fit for CT type applications as it more closely follows the CT HTTP API.
+
+The server now returns 403 for permission denied where it used to return 500 errors. This follows the behaviour of the C++ implementation.
+
+The log signer binary now reports metrics for the number it has signed and the number of errors that have occurred. This is intended to give more insight into the state of the queue and integration processing.
+
+Commit [b20b3109af7b68227c83c5d930271eaa4f0be771](https://api.github.com/repos/google/trillian/commits/b20b3109af7b68227c83c5d930271eaa4f0be771) Download [zip](https://api.github.com/repos/google/trillian/zipball/v1.0.6)
+
+## v1.0.5 - TLS, Merge Delay Metrics, Easier Admin Tests
+
+Published 2018-02-05 15:54:26 +0000 UTC
+
+The API protos have been rebuilt with GRPC 1.3.
+
+Timestamps have been added to the log leafs in the MySQL database. Before upgrading to this version you **must** make the following schema changes:
+
+Add the following column to the `LeafData` table. If you have existing data in the queue you might have to remove the NOT NULL clause:
+
+`QueueTimestampNanos  BIGINT NOT NULL`
+
+Add the following column to the `SequencedLeafData` table:
+
+`IntegrateTimestampNanos BIGINT NOT NULL`

```

Comment:
Maybe make the 4 lines above into bullets + code blocks to make the grouping clearer:

 - Add the following column to the `LeafData` table. If you have existing data in the queue you might have to remove the `NOT NULL` clause:
```QueueTimestampNanos  BIGINT NOT NULL```
 - Add the following column to the `SequencedLeafData` table:
￼ ```IntegrateTimestampNanos BIGINT NOT NULL```

Reply:
Done.